+++
title = "apex: phylogenetics with multiple genes"
date = "2017-01-01"
draft = false
abstract = "Genetic sequences of multiple genes are becoming increasingly common for a wide range of organisms including viruses, bacteria and eukaryotes. While such data may sometimes be treated as a single locus, in practice, a number of biological and statistical phenomena can lead to phylogenetic incongruence. In such cases, different loci should, at least as a preliminary step, be examined and analysed separately. The r software has become a popular platform for phylogenetics, with several packages implementing distance-based, parsimony and likelihood-based phylogenetic reconstruction, and an even greater number of packages implementing phylogenetic comparative methods. Unfortunately, basic data structures and tools for analysing multiple genes have so far been lacking, thereby limiting potential for investigating phylogenetic incongruence. In this study, we introduce the new r package apex to fill this gap. apex implements new object classes, which extend existing standards for storing DNA and amino acid sequences, and provides a number of convenient tools for handling, visualizing and analysing these data. In this study, we introduce the main features of the package and illustrate its functionalities through the analysis of a simple data set."
authors = ["Thibaut Jombart", "Frederick Archer", "Klaus Schliep", "Zhian Kamvar", "Rebecca Harris", "Emmanuel Paradis", "JÃ©rome Goudet", "Hilmar Lapp"]
publication_types = ["2"]
publication = "In *Molecular Ecology Resources*"
publication_short = ""
image_preview = ""
selected = false
projects = []
tags = []
url_pdf = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
math = false
highlight = true
[header]
image = ""
caption = ""
[[url_custom]]
name = "View Journal Article"
url = "http://dx.doi.org/10.1111/1755-0998.12567"
+++
